packages/scripts/webapp/scripts/start.js

```diff
@@ -3,16 +3,18 @@ const { hereRelative, resolveBin } = require('../utils/path-resolver');
 
 const webpackDevServer = resolveBin('webpack-dev-server');
 
-module.exports = function start(env) {
+module.exports = function start(env, _, options) {
 	return spawn.sync(
 		webpackDevServer,
 		[
 			'--inline',
-			'--config', hereRelative(__dirname, '../config/webpack.config.js'),
-			'--content-base', 'build/',
```

Comment:
Just double checking --content-base was removed on purpose?

Reply:
Yes, in fact the path is wrong. Let's manage that in the devServer config